docs/consensus_decisions.md

```diff
@@ -0,0 +1,51 @@
+# Consensus Decisions
+
+During its proceedings, the working group has reached internal consensus on a number of issues.
+This document enumerates those, and provides a reference for later actions.
+
+## 1: Include message references in the data model.
+
+**Discussion:**
+The implementers would find a way to include references anyways, but including it in the data model (standard) can make it subject to best practices.
+It’s still possible for users to do “the wrong thing” (ex: concatenation of strings/messages), but then you would find it more difficult to achieve.
+
+One of the drawbacks of message references is that referenced messages effectively have a public API (names of parameters, variables, variants, etc.) which must be consistent across all callsites.
+This leads us to consensus 2.
+
+## 2: Allow parameters passed with message references to the message being referenced and validate it.
```

Comment:
This is a little difficult for me to parse. Could we say "Allow referencing and validation of parameters passed with message references to the message"?

Reply:
```suggestion
## 2: Allow message references to include parameters in a form that enables their validation.
```

Would that be clearer?

Reply:
Yes, that looks good to me.